src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -47,6 +47,12 @@
             "name": "detectorId",
             "type": "string",
             "description": "The id of the detector which produced the code scan issue"
+        },
+        {
+            "name": "applyFixSource",
+            "type": "string",
+            "allowedValues": ["hover", "webview", "quickfix"],
```

Comment:
you could add these values to the existing `component` type: https://github.com/aws/aws-toolkit-common/blob/bbc34c2f5b5f1f0a24befa5c42f4193e3e12032c/telemetry/definitions/commonDefinitions.json#L57

then just use `component` in `codewhisperer_codeScanIssueApplyFix`